.travis.yml

```diff
@@ -1,27 +1,98 @@
 language: rust
+sudo: false
 branches:
   only:
     - master
-os:
-  - linux
-  - windows
-rust:
-  - stable
-  - beta
-  - nightly
+
 # Need to cache the whole `.cargo` directory to keep .crates.toml for
 # cargo-update to work, but don't cache the cargo registry
 before_cache:
   - rm -rf /home/travis/.cargo/registry
 cache:
   directories:
     - /home/travis/.cargo
-before_script:
-  - if [[ "$TRAVIS_RUST_VERSION" == "stable" || "$TRAVIS_RUST_VERSION" == "nightly" ]]; then rustup component add clippy-preview; fi
-script:
-  - if [[ "$TRAVIS_RUST_VERSION" == "stable" || "$TRAVIS_RUST_VERSION" == "nightly" ]]; then cargo clippy --all-targets --all-features -- -D warnings; fi
-  - cargo test --verbose
+
+DEPLOY_TO_GITHUB: &DEPLOY_TO_GITHUB
+  before_deploy:
+    - git config --local user.name "Stuart Crust"
+    - git config --local user.email "stuart.crust@gmail.com"
```

Comment:
Is the `git config` necessary if Travis isn't committing on my behalf?

Reply:
Good question - I'm not actually sure 😅 I followed the convention that Ash had done for [`rust-wasm`](https://github.com/rustwasm/wasm-pack/blob/51e6351c28fbd40745719e6d4a7bf26dadd30c85/.travis.yml#L22-L23) and I can't find anything in the docs for [GH Releases](https://help.github.com/articles/about-releases/). Gunna remove this for now